packages/lesswrong/lib/collections/tags/fragments.ts

```diff
@@ -3,11 +3,13 @@ import { registerFragment } from '../../vulcan-lib';
 registerFragment(`
   fragment TagBasicInfo on Tag {
     _id
+    userId
```

Comment:
Can now remove?

Reply:
ah yeah we can, I'll make a note to do this in one of my other PRs though

Reply:
Do try to instill a habit of reading the comments before merging, even on approval.